crates/ruff_linter/src/rules/airflow/rules/removal_in_3.rs

```diff
@@ -154,6 +155,112 @@ fn removed_argument(checker: &mut Checker, qualname: &QualifiedName, arguments:
     };
 }
 
+fn diagnostic_for_method(deprecated: &Identifier, replacement: Replacement) -> Option<Diagnostic> {
+    let diagnostic = Diagnostic::new(
+        Airflow3Removal {
+            deprecated: deprecated.as_str().to_string(),
+            replacement,
+        },
+        deprecated.range(),
+    );
+
+    Some(diagnostic)
+}
+
+fn removed_method(checker: &mut Checker, func: &Box<Expr>) {
```

Comment:
Clippy suggest me to change it to `&Expr`, but it failed after I changed it. Would like to see whether there's suggestion on this one. Thanks!